From 0b022e0b76ace2b484d7c375a6b8b1ee6d21fc20 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jakub=20Je=C4=8Dm=C3=ADnek?= <kuba@kubajecminek.cz>
Date: Thu, 29 Feb 2024 23:04:38 +0100
Subject: [PATCH] Adjust config

---
 config.def.h | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/config.def.h b/config.def.h
index d805331..d256020 100644
--- a/config.def.h
+++ b/config.def.h
@@ -65,5 +65,10 @@ static const char unknown_str[] = "n/a";
  */
 static const struct arg args[] = {
 	/* function format          argument */
-	{ datetime, "%s",           "%F %T" },
+	{ datetime,      " %s | ",                 "%d.%m.%Y %T" },
+	{ disk_perc,     "Disk: %s % | ",          "/"           },
+	{ cpu_perc,      "CPU: %s % | ",           NULL          },
+	{ ram_perc,      "RAM: %s % | ",           NULL          },
+	{ wifi_essid,    "WIFI: %s | ",            "wlp2s0"      },
+	{ battery_perc,  "Battery: %s %   ",       "BAT0"        },
 };
-- 
2.42.0

