From aa86b5f078993ad862051cd6bc3f227f4fca5223 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jakub=20Je=C4=8Dm=C3=ADnek?= <kuba@kubajecminek.cz>
Date: Fri, 16 Feb 2024 16:05:58 +0100
Subject: [PATCH] Adjust config.def.h

---
 config.def.h | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/config.def.h b/config.def.h
index 061ad66..e54d67b 100644
--- a/config.def.h
+++ b/config.def.h
@@ -7,11 +7,11 @@ static const int showbar            = 1;        /* 0 means no bar */
 static const int topbar             = 1;        /* 0 means bottom bar */
 static const char *fonts[]          = { "monospace:size=10" };
 static const char dmenufont[]       = "monospace:size=10";
-static const char col_gray1[]       = "#222222";
-static const char col_gray2[]       = "#444444";
-static const char col_gray3[]       = "#bbbbbb";
-static const char col_gray4[]       = "#eeeeee";
-static const char col_cyan[]        = "#005577";
+static const char col_gray1[]       = "#191724";
+static const char col_gray2[]       = "#4c566a";
+static const char col_gray3[]       = "#908caa";
+static const char col_gray4[]       = "#e0def4";
+static const char col_cyan[]        = "#eb6f92";
 static const char *colors[][3]      = {
 	/*               fg         bg         border   */
 	[SchemeNorm] = { col_gray3, col_gray1, col_gray2 },
@@ -45,7 +45,7 @@ static const Layout layouts[] = {
 };
 
 /* key definitions */
-#define MODKEY Mod1Mask
+#define MODKEY Mod4Mask
 #define TAGKEYS(KEY,TAG) \
 	{ MODKEY,                       KEY,      view,           {.ui = 1 << TAG} }, \
 	{ MODKEY|ControlMask,           KEY,      toggleview,     {.ui = 1 << TAG} }, \
-- 
2.42.0

